src/tasks/component-installers/devfile-workspace-operator-installer.ts

```diff
@@ -273,6 +282,35 @@ export class DevWorkspaceTasks {
           task.title = await `${task.title}...OK`
         }
       },
+      {
+        title: `Delete the Custom Resource of type ${this.cheManagerCRDName}`,
+        task: async (_ctx: any, task: any) => {
+          await this.kubeHelper.deleteCheManagerInstance(this.devworkspaceCheNamespace)
+          // if chemanager instance still exists then remove finalizers and delete again
+          const chemanager = await this.kubeHelper.getCheManagerInstance(this.devworkspaceCheNamespace)
```

Comment:
We need a pause between delete and get.
Since deletion means sending request to delete an object, so object will be removed after a while.

Reply:
Got it.